src/Relation/Binary/Construct/Closure/Reflexive/Properties.agda

```diff
@@ -0,0 +1,59 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties of reflexive closures
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K #-}
+
+module Relation.Binary.Construct.Closure.Reflexive.Properties where
+
+open import Data.Product as Prod
+open import Function
+open import Relation.Binary
+open import Relation.Binary.Construct.Closure.Reflexive
+import Relation.Binary.PropositionalEquality as PropEq
+open PropEq using (_≡_; refl)
```

Comment:
No need to split the `import` and `open` statements here

Reply:
Unsplit